grai-integrations/source-bigquery/src/grai_source_bigquery/models.py

```diff
@@ -144,6 +144,7 @@ class Constraint(str, Enum):
     foreign_key = "f"
     primary_key = "p"
     belongs_to = "bt"
+    bigquery_model = "bqm"
```

Comment:
What do bigquery model constraints represent?

Reply:
These are table to table edges from the logs, see the dbt equivalent

Reply:
We should get together and try to standardize these in some capacity. It seems like a hassle to have to change the web app for every new integration.

Reply:
Strangely I don't think these actually go anywhere, because I think when the `Edge` models get created they get inspected as `TableToTable` based on having table sources and destinations.